packages/components/src/tooltip/index.js

```diff
@@ -111,6 +129,15 @@ function Tooltip( props ) {
 	const [ isOver, setIsOver ] = useState( false );
 	const delayedSetIsOver = useDebounce( setIsOver, delay );
 
+	// Create a reference to the Tooltip's child, to be passed to the Popover
+	// so that the Tooltip can be correctly positioned. Also, merge with the
+	// existing ref for the first child, so that its ref is preserved.
+	const childRef = useRef( null );
+	const mergedChildRefs = useMergeRefs( [
+		childRef,
+		Children.toArray( children )[ 0 ]?.ref,
```

Comment:
Can you not use `child` here? That would make things a bit clearer.

Reply:
Replied in https://github.com/WordPress/gutenberg/pull/41268#discussion_r920743047